examples/nlp/question_answering_squad.py

```diff
@@ -16,12 +16,14 @@
 
 Some transformer of this code were adapted from the HuggingFace library at
 https://github.com/huggingface/transformers
+"""
 
+"""
 Download the Squad data by running the script:
-examples/nlp/scripts/download_squad.py
```

Comment:
why? is *get_aaa* better than *download_aaa*?

Reply:
would be addressed in next stage.